vms/platformvm/tx.go

```diff
@@ -108,7 +111,11 @@ type Tx struct {
 }
 
 // Sign this transaction with the provided signers
-func (tx *Tx) Sign(c codec.Manager, signers [][]*crypto.PrivateKeySECP256K1R) error {
+func (tx *Tx) Sign(
```

Comment:
I disagree on this solution. Changing sign requires changes across all tx. Is there a reason we can't just extend the Tx interface with a new function 'SignWithRsaPrivateKey' and encapsulate the change. Not that I think it matters here a lot but in general this will help us to merge more easily with Avalanche in the future.

Reply:
Just for history:
It was made this way because of `tx.Initialize` at the bottom of the `Sign` function. `Initialize` requires signed bytes.
We discussed it and decided that its better to call `Initialize` twice (2nd call will overwrite 1st one) in 2 different `Sign` methods, than to have changes in `Sign` method signature.